features/claim-mgt/org.wso2.carbon.claim.mgt.server.feature/resources/conf/claim-config.xml

```diff
@@ -595,6 +595,24 @@
 				<AttributeID>active</AttributeID>
 				<Description>Status of the account</Description>
 			</Claim>
+			<Claim>
+				<ClaimURI>http://wso2.org/claims/identity/secretkey</ClaimURI>
+				<DisplayName>Secret Key</DisplayName>
+				<AttributeID>totpSecretkey</AttributeID>
+				<Description>Claim to store the secret key</Description>
+			</Claim>
+			<Claim>
+				<ClaimURI>http://wso2.org/claims/identity/emailotp_disabled</ClaimURI>
```

Comment:
Shouldn't we follow the camel case convention for all claims?

Reply:
Created a git issue for this https://github.com/wso2/product-is/issues/6486